### ISSUE TITLE
Public Comment on 2.3.3 Animation from Interactions and prefers-reduced-motion

### ISSUE THREAD

**Issue by CharlesBelov** [w3c/wcag3#183]
Since WCAG 2.1 was finalized, a new technology has emerged in which some operating systems expose to user agents, and user agents in turn expose to content models, that the user has requested animation be "reduced" (Mac, iOS wording) or eliminated (Windows).

My comment is that if a user has requested this reduction/elimination and the user agent exposes that information to the content, respecting that request by not doing any animation from interaction would appropriately be AA. 

As I recall, the concern with making item 2.3.3 be AA with WCAG 2.1 was that there was insufficient study to decide whether a particular animation is problematic. I believe that the user telling their OS that animated content is problematic is sufficient to determine that it is unnecessary to make the problematic-or-not determination, and just eliminate all non-essential animation in such cases.

This is already an emerging best practice and would best be formalized as part of WCAG 2.2.

If an operating system or user agent does not pass this information through to the content, the AAA specification would remain as it is in WCAG 2.1.
